CMake Error at ports/zziplib/portfile.cmake:35 (file):
  file RENAME failed to rename

    D:/p/zziplib_x86-windows/share/pkgconfig/zzipfseeko.pc

  to

    D:/p/zziplib_x86-windows/lib/pkgconfig/zzipfseeko.pc

  because: The system cannot find the path specified.

--- a/ports/zziplib/portfile.cmake
+++ b/ports/zziplib/portfile.cmake
@@ -5,6 +5,7 @@ vcpkg_from_github(
     SHA512 bed63fa7d430bd197bb70084f28ae6edc4c4120655b882bc8367f968b32c03340bb6d9bf1b14a5fcc5a1160d91ccf00e7b1131a4123da5d52233a84840ba8b7e
     PATCHES
         no-release-postfix.patch
+        revert-pkgconfig-path.patch
 )
 
 string(COMPARE EQUAL VCPKG_CRT_LINKAGE "static" MSVC_STATIC_RUNTIME)
@@ -31,15 +32,12 @@ vcpkg_cmake_install()
 vcpkg_cmake_config_fixup(CONFIG_PATH "lib/cmake/zziplib")
 vcpkg_fixup_pkgconfig()
 
-file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/lib/pkgconfig")
-file(RENAME "${CURRENT_PACKAGES_DIR}/share/pkgconfig/zzipfseeko.pc" "${CURRENT_PACKAGES_DIR}/lib/pkgconfig/zzipfseeko.pc")
-file(RENAME "${CURRENT_PACKAGES_DIR}/share/pkgconfig/zziplib.pc" "${CURRENT_PACKAGES_DIR}/lib/pkgconfig/zziplib.pc")
-file(RENAME "${CURRENT_PACKAGES_DIR}/share/pkgconfig/zzipmmapped.pc" "${CURRENT_PACKAGES_DIR}/lib/pkgconfig/zzipmmapped.pc")
-
 file(REMOVE_RECURSE
     "${CURRENT_PACKAGES_DIR}/debug/include"
-    "${CURRENT_PACKAGES_DIR}/debug/share"
-    "${CURRENT_PACKAGES_DIR}/share/pkgconfig"
+    "${CURRENT_PACKAGES_DIR}/debug/lib/pkgconfig/zzipfseeko.pc"
+    "${CURRENT_PACKAGES_DIR}/debug/lib/pkgconfig/zzipmmapped.pc"
+    "${CURRENT_PACKAGES_DIR}/lib/pkgconfig/zzipfseeko.pc"
+    "${CURRENT_PACKAGES_DIR}/lib/pkgconfig/zzipmmapped.pc"
 )
 
 file(STRINGS "${CURRENT_PACKAGES_DIR}/include/zzip/_config.h" have_stdint_h REGEX "^#define ZZIP_HAVE_STDINT_H 1")

--- a/ports/zziplib/revert-pkgconfig-path.patch
+++ b/ports/zziplib/revert-pkgconfig-path.patch
@@ -1,0 +1,13 @@
+diff --git a/zzip/CMakeLists.txt b/zzip/CMakeLists.txt
+index 646903b..c300011 100644
+--- a/zzip/CMakeLists.txt
++++ b/zzip/CMakeLists.txt
+@@ -304,7 +304,7 @@ set(outdir ${CMAKE_CURRENT_BINARY_DIR})
+ 
+ if(ZZIP_PKGCONFIG)
+ install(FILES ${outdir}/zziplib.pc ${outdir}/zzipmmapped.pc ${outdir}/zzipfseeko.pc
+-        DESTINATION "${CMAKE_INSTALL_DATADIR}/pkgconfig" )
++        DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig" )
+ endif()
+ 
+ install(FILES ${libzzip_HDRS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/zzip )

--- a/versions/z-/zziplib.json
+++ b/versions/z-/zziplib.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "c891bfe5dcf8f903a1bdc521dadabb1d3cbb680a",
+      "git-tree": "1c877d218b88327087916afa94f68fd0ba8fbc00",
       "version": "0.13.79",
       "port-version": 0
     },

